v4l2/v4l2_codecbase.h

```diff
@@ -96,6 +98,8 @@ class V4l2CodecBase {
     bool m_started;
 #if __ENABLE_V4L2_GLX__
     Display *m_x11Display;
+#else
+    int m_drmfd;
```

Comment:
we need init this in V4l2CodecBase's constructor. Just as we did to m_x11Display